src/atoms/weavemol.jl

```diff
@@ -0,0 +1,13 @@
+# some commentary
+
+# docstring...
+mutable struct WeaveMol <: AbstractAtoms
+    smiles::String
+    elements::Vector{String}
+    atom_features::Vector{SomethingOrOther} # I need to look more carefully to figure this out, heh
+    pair_features::Vector{SomethingOrOther}
+    featurization::WeaveFeaturization
```

Comment:
Would this be better as `Union{WeaveFeaturization, Nothing}`?

Reply:
yup